pkg/sip/inbound.go

```diff
@@ -128,6 +128,16 @@ func (s *Server) onInvite(log *slog.Logger, req *sip.Request, tx sip.ServerTrans
 	_ = s.processInvite(req, tx)
 }
 
+func (s *Server) handleSessionEnd(ctx context.Context, callInfo *rpc.SIPCall, state *CallState, sipCallID string, reason string, projectID string) {
+	if s.handler != nil {
+		go s.handler.OnSessionEnd(context.Background(), &CallIdentifier{
+			ProjectID: projectID,
+			CallID:    callInfo.LkCallId,
+			SipCallID: sipCallID,
+		}, state.callInfo, reason)
```

Comment:
This might race on `state.callInfo`, I guess.

Reply:
We pass that along in case it could be used for any post processing and expect it to be read only. So I think we are good here.